packages/framework/store/src/transformer/job.ts

```diff
@@ -470,6 +470,11 @@ export class Job {
       children,
     });
 
+    const nextTick =
+      typeof window !== 'undefined'
+        ? window.requestAnimationFrame
```

Comment:
This is considered too slow in #8380. A brief perf test could be found in

https://github.com/toeverything/blocksuite/pull/8427#issuecomment-2365224310

The non-blocking pasting could be MUCH slower.

Try this test file with ~1000 lines, the sync pasting completes immediately but async pasting takes ~10s.

[clipboard-test.md](https://github.com/user-attachments/files/17084690/clipboard-test.md)